slices/erlang-crypto.yaml

```diff
@@ -0,0 +1,18 @@
+package: erlang-crypto
+
+essential:
+  - erlang-crypto_copyright
+
+slices:
+  libs:
+    essential:
+      - erlang-base_bins
```

Comment:
why do these libs need the erland-base_bins?

Reply:
see comment above.

Reply:
https://github.com/canonical/chisel-releases/pull/258#discussion_r1802799227